Core/Core/Kits/Applications.cs

```diff
@@ -64,6 +64,7 @@ public static class HostApplications
     Unity = new("Unity", "unity"),
     GSA = new("GSA", "gsa"),
     Civil = new("Civil 3D", "civil3d"),
```

Comment:
do we need this one if we added Civil3D?

Reply:
Yes, to prevent breaking the DUI2 connectors

Reply:
I believe we should start to consider have new Applications(?) class for DUI3 only instead populating old.

Reply:
FYI: @BovineOx added as subtask under the other relevant story (assigned to Ian) https://spockle.atlassian.net/browse/DUI3-431